src/app.html

```diff
@@ -135,6 +136,16 @@
 				Footer content
 			</span> -->
 		</div>
+		<script>
+			if ('serviceWorker' in navigator) {
```

Comment:
I'm not entirely sure if this part is necessary, afaik introducing serviceWorker will cache the html, js, css files and won't be updated for subsequent releases unless we explicitly add an option to remove cache.

Reply:
There's ways to bust the cache. Typically I've done it by appending a timestamp to the file names, and updating it for an updated release.

Reply:
Too much of a hassle to include imo, ROI seems pretty low for this one. Besides PWA already works as intended on IOS, Android devices.